### DISCUSSION TITLE
copilot.vim default copilot-next and copilot-previous not working on macOS

### DISCUSSION THREAD

**Discussion by dzfrias** [community/community#19376] [Copilot Conversations]
## Problem
In the neovim plugin for GitHub copilot, [copilot.vim](https://github.com/github/copilot.vim), the default bindings for `copilot-next` and `copilot-previous` do not work on macOS. This is caused by the incompatibility between the `<A>` (alt) modifier and macOS.

## Fix
As a user, it is fixable with:
```vim
imap ‘ <Plug>(copilot-next)
imap “ <Plug>(copilot-previous)
```

And could be fixed easily by changing a few default bindings if `has('macunix')` is true.

I'm not sure how long the alt modifier incompatibility has been around (it could be due to a recent macOS software update) so some further system version checking might be needed in order for copilot.vim to work across all macOS versions.

## Update
This depends on user settings, so there isn't much a plugin can do to fix this problem for all users of a given platform.

**Comment by jsgv** [comment]
How would you type the actual `'` & `"` characters, or do you simply never use them?
